starwars/src/components/Card.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import CardHeader from './CardHeader';
+import CardContent from './CardContent';
+
+const Card = props => {
+    return (
+        <div>
```

Comment:
remember what we discussed in our zoom: this div is responsible for containing the cards, so this is where we want a className that adds `display: flex` and `flex-wrap: wrap`